### PR TITLE
Remove extern(Pascal), deprecated in v2.084.0

### PR DESCRIPTION
**Pull request by Geod24**
It's been 10 releases, time to go! I could make it an explicit error but I don't think that makes much sense.
Will raise dlang.org PR.

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

**Review by ibuclaw — APPROVED** [review]

**Comment by Geod24** [comment]
dlang.org was already cleaned up when I did the deprecation, there's only the code in druntime left: https://github.com/dlang/druntime/pull/3285

**Comment by Geod24** [comment]
I changed one of the unit tests. Previously it was simply adapted to use user-provided deprecation, however it's a parser test, not a semantic test, so deprecations won't get triggered. Any deprecation in the parser is likely to be temporary, so it would just be pushing the problem to the next contributor for dubious reasons, hence why I just removed it.

File: src/dmd/frontend.h [hunk 1]
```diff
@@ -171,9 +171,8 @@ enum class LINK : uint8_t
     c = 2u,
     cpp = 3u,
     windows = 4u,
-    pascal = 5u,
-    objc = 6u,
-    system = 7u,
+    objc = 5u,
```
Comment (ibuclaw) [inline src/dmd/frontend.h:174]: Oops, just realised that you broke downstream gdc and ldc.

Comment (Geod24) [reply src/dmd/frontend.h:174]: Isn't it expected tho ?

Comment (ibuclaw) [reply src/dmd/frontend.h:174]: > Isn't it expected tho ?

You didn't update global.h

Comment (Geod24) [reply src/dmd/frontend.h:174]: Ergh, an oversight, will fix.

**Pull request merged**